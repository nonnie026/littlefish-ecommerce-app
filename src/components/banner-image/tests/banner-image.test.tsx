import { render } from '@testing-library/react'
import { BannerImage } from '../index'

describe('BannerImage', () => {
  it('renders correctly and matches snapshot', () => {
    const { asFragment } = render(<BannerImage />)

    expect(asFragment()).toMatchSnapshot()
  })
})
