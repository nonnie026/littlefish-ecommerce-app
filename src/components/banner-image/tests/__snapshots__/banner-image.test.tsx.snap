// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BannerImage renders correctly and matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root css-l5bi8q"
  >
    <img
      alt="banner"
      class="MuiBox-root css-6jrdpz"
      src=""
    />
  </div>
</DocumentFragment>
`;
